src/Filters/SingleCheckbox.jsx

```diff
@@ -3,12 +3,12 @@ import React from 'react';
 export default class SingleCheckbox extends React.Component {
   render() {
     return (
-      <section className="single-checkbox">
```

Comment:
I think I found where I can change the HTML/JSX quotes.  Will resubmit because I agree with ypu